modules/assim.batch/R/pda.postprocess.R

```diff
@@ -0,0 +1,192 @@
+##' Postprocessing for PDA Results
+##'
+##' @title Postprocessing for PDA Results
+##' @param all params are the identically named variables in pda.mcmc / pda.emulator
+##'
+##' @return PEcAn settings list, updated with <params.id> pointing to the new params file.
+##'
+##' @author Ryan Kelly, Istem Fer
+##' @export
+pda.postprocess <- function(settings, con, mcmc.param.list, pname, prior, prior.ind) {
+  
+  params.subset <- pda.plot.params(settings, mcmc.param.list, prior.ind)
+  
+  for (i in seq_along(settings$pfts)) {
+    
+    ## Save params
+    filename.mcmc <- file.path(settings$pfts[[i]]$outdir, 
+                               paste0("mcmc.pda.", 
+                                      settings$pfts[[i]]$name, 
+                                      "_", 
+                                      settings$assim.batch$ensemble.id, 
+                                      ".Rdata"))
+    params.pft <- params.subset[[i]]
+    save(params.pft, file = filename.mcmc)
+    
+    ## create a new Posteriors DB entry
+    now <- format(Sys.time(), "%Y-%m-%d %H:%M:%S")
+    
+    pft.id <- db.query(paste0("SELECT pfts.id FROM pfts, modeltypes WHERE pfts.name='",
+                              settings$pfts[[i]]$name, 
+                              "' and pfts.modeltype_id=modeltypes.id and modeltypes.name='", 
+                              settings$model$type, "'"), 
+                       con)[["id"]]
+    
+    db.query(paste0("INSERT INTO posteriors (pft_id, created_at, updated_at) VALUES (",
+                    pft.id, ", '", now, "', '", now, "')"), con)
+    
+    posteriorid <- db.query(paste0("SELECT id FROM posteriors WHERE pft_id=", pft.id, " AND created_at='", 
```

Comment:
@mdietze @bcow in this bit of code I'm inserting a posterior to the DB and need the `id` of the inserted posterior, if I don't insert it with `created_at`, how can I retrieve it back here?

currently I was doing this:
`     posteriorid <- db.query(paste0("SELECT id FROM posteriors WHERE pft_id=", pft.id, " AND created_at='",  now, "'"), con)[["id"]]`

I don't see any other column in the posterior table. I can filter for the latest created posterior but would that be prone to errors?